accounts/tests/tests.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from django.contrib.auth.models import User
+from django.template.loader import get_template
+
+from house.constants import USER_LOGIN_PAGE_ROUTE, USER_SIGNUP_ROUTE
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "TestPassword123!@#"
+PASSWORD_NOT_MATCH = "NOT_MATCH"
+EMAIL = "testuser@gmail.com"
+
+
+@pytest.fixture
+def new_user():
+    user = User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, password=PASSWORD, email=EMAIL)
+    user.set_password(PASSWORD)
+    user.save()
+    return user
+
+
+@pytest.mark.django_db()
+class TestLoginPage:
+    def test_user_login_template(self):
+        get_template(USER_LOGIN_PAGE_ROUTE)
+
+    def test_login_entry_point(self, client):
+        response = client.get('/accounts/login/')
+        assert response.status_code == 200
+        assert 'registration/login.html' in response.template_name
+
+    def test_login_user(self, client, new_user):
+        response = client.post('/accounts/login/', {'username': USERNAME, 'password': PASSWORD})
+        assert response.status_code == 302
+        assert response.url == '/../house/'
+
+    def test_bad_login(self, client):
+        response = client.post('/accounts/login/', {'username': USERNAME, 'password': PASSWORD})
+        assert response.status_code == 200
+        assert 'registration/login.html' in response.template_name
```

Comment:
How is this a bad login? it seems like the previous test

Reply:
User never created so the login will fail